js/main.js

```diff
@@ -1,2 +1,45 @@
 /* global data */
 /* exported data */
+
+var $urlInput = document.querySelector('.url');
+var $imageEntry = document.querySelector('.image-entry');
+var $entryForm = document.querySelector('#journal-entry-form');
+
+function addPhoto(event) {
+  // console.log(urlInput.value);
+  $imageEntry.setAttribute('src', $urlInput.value);
+
+}
+
+$urlInput.addEventListener('input', addPhoto);
+
+function submitInfo(event) {
+  // event.preventDefault();
+  var formInfoObj = {};
+  // console.log('Submitted');
+  // console.log($entryForm.elements);
+
+  formInfoObj.title = $entryForm.elements.title.value;
+  formInfoObj.url = $entryForm.elements.url.value;
+  formInfoObj.notes = $entryForm.elements.notes.value;
+  formInfoObj.nextEntryId = data.nextEntryId;
+
+  data.entries.unshift(formInfoObj);
+  data.nextEntryId++;
+  var stringifyObj = JSON.stringify(data);
+  localStorage.setItem('localData', stringifyObj);
```

Comment:
Instead of manually stringifying and storing the data model here, try setting up a `beforeunload` like in our local storage exercise.